lib/LTP/kirk.pm

```diff
@@ -0,0 +1,120 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Executes kirk testing framework
+# Maintainer: Kernel QE <kernel-qa@suse.de>
+
+package LTP::kirk;
+
+use strict;
+use warnings;
+use Exporter;
+use utils;
+use testapi;
+use testapi qw(get_var get_required_var);
+use serial_terminal 'select_serial_terminal';
+use version_utils qw(is_transactional);
+use transactional 'trup_install';
+use LTP::utils;
+
+sub _kirk_from_git
```

Comment:
`_` ? perl !=python

Reply:
I seen it's somehow a convention to define "private" methods which are not called outside. `LTP/WhiteList.pm` is also using it: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/b93e7ae6585a105f1be7720801454c7df580e558/lib/LTP/WhiteList.pm#L40